src/context/TransactionsProvider/TransactionsProvider.tsx

```diff
@@ -209,7 +210,12 @@ export const TransactionsProvider: React.FC<TransactionsProviderProps> = ({ chil
         // subscribe to new transactions for all supported accounts
         try {
           return adapter?.subscribeTxs(
-            { wallet, accountType, accountNumber },
+            {
+              wallet,
+              accountType,
+              accountNumber,
+              pubKey: isLedger(wallet) && accountId ? fromAccountId(accountId).account : undefined,
```

Comment:
see https://github.com/shapeshift/web/pull/5444/files#r1357602957

Reply:
Same rationale as https://github.com/shapeshift/web/pull/5444/files#r1357880836 - we probably don't want to *consistently* use unchained, only for Ledger where it's actually a functional requirement